geode-core/src/main/java/org/apache/geode/internal/cache/wan/serial/SerialGatewaySenderEventProcessor.java

```diff
@@ -863,7 +863,7 @@ public void sendBatchDestroyOperationForDroppedEvent(EntryEventImpl dropEvent, i
     } catch (Exception e) {
       if (logger.isDebugEnabled()) {
         logger.debug(
-            "Exception in sending dropped event could be ignored in order not to interrupt sender starting",
+            "Exception in sending dropped event could be ignored in order not to interrupt sender starting: {}",
```

Comment:
I think e.toString needs to be added otherwise it translates it to missing parameter as it translates the method to debug(String message, Throwable ex)

Reply:
I will leave it as it was as I did not know that the method could take a Throwable as a second parameter.